spdata/common.py

```diff
@@ -0,0 +1,24 @@
+"""Constants and types common for all modules in package
+
+Copyright 2018 Spectre Team
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+    
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import os
+
+_FILESYSTEM_ROOT = os.path.abspath(os.sep)
+DATA_ROOT = os.path.join(_FILESYSTEM_ROOT, 'data')
+
+Name = str
+Path = str
```

Comment:
I know this is huge to refactor, but shouldn’t we utilize [pathlib](https://docs.python.org/3/library/pathlib.html) for working with all paths? Aside from standardized way of checking for file existence, directory tests, we get a first-class citizen type `Path` we could pass to dataset loaders.

Reply:
That's a biggun. Gonna consider it but probably not now.